From 007ac5c96eba48f86cf562092ea10dcc244a0f1f Mon Sep 17 00:00:00 2001
From: Ran Wang <ran.wang_1@nxp.com>
Date: Wed, 13 Jun 2018 15:47:47 +0800
Subject: [PATCH 063/107] Revert "powerpc/p1_p2_rdb_pc: Fix endian access issue on EHCI intinalization"

There was an EHCI endian accessor issue. Now it's fixed by
commit 9829ce2ff25c ("usb: ehci: Fix accessors for big-endian platforms and descriptors").
Revert commit 0f2296bab141 ("powerpc/p1_p2_rdb_pc: Fix endian access issue on EHCI intinalization")
due to it fix that issue in a wrong way. Actually, on P1 P2 platforms,
USB EHCI register use little endian.

Signed-off-by: Ran Wang <ran.wang_1@nxp.com>
---
 include/configs/p1_p2_rdb_pc.h |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/include/configs/p1_p2_rdb_pc.h b/include/configs/p1_p2_rdb_pc.h
index 624f9eb..180a1eb 100644
--- a/include/configs/p1_p2_rdb_pc.h
+++ b/include/configs/p1_p2_rdb_pc.h
@@ -771,7 +771,6 @@
 #ifdef CONFIG_USB_EHCI_HCD
 #define CONFIG_EHCI_HCD_INIT_AFTER_RESET
 #define CONFIG_USB_EHCI_FSL
-#define CONFIG_EHCI_DESC_BIG_ENDIAN
 #endif
 #endif
 
-- 
1.7.1

